openapi_test.go

```diff
@@ -2,28 +2,48 @@ package fuego
 
 import (
 	"bytes"
+	"encoding/xml"
 	"fmt"
 	"io"
+	"log/slog"
 	"net/http"
 	"net/http/httptest"
 	"os"
 	"testing"
 
 	"github.com/getkin/kin-openapi/openapi3"
 	"github.com/stretchr/testify/require"
+	"github.com/thejerf/slogassert"
+	"gotest.tools/v3/assert"
```

Comment:
I think you wanted to use testify/assert and your IDE provided a random lib

Reply:
Updated. I believe this is because gotest is in our base go.mod do to the petstore example not being it's own module. I'll be fixing that in the another PR